examples/service/stateful/providers/azure/importVm/main.go

```diff
@@ -51,6 +51,7 @@ func main() {
 								TagValue: spotinst.String("bar"),
 							},
 						},
+						UserData: spotinst.String("VGhpcyBpcyBhIHZhbGlkIGJhc2UgNjQgdXNlciBkYXRhIHNjcmlwdA=="),
```

Comment:
update as userData instead of actual value

Reply:
done